src/components/Home/Home.js

```diff
@@ -77,61 +80,61 @@ class Home extends React.Component {
   }
 
   render() {
+    console.log('hey');
```

Comment:
```suggestion
```

Reply:
🤦‍♂ not the best way definetily, but the quicker :-P
Thanks for noticing!